configure.ac

```diff
@@ -782,10 +782,10 @@ fi
 AC_OUTPUT
 
 AS_IF([test "$enable_pmi_mpi" != "yes" -a "$enable_pmi_simple" != "yes" -a "$opal_external_pmix_version_found" != 1 -a "$ompi_check_pmi_happy" != "yes" -a -z "$pmi_type"],
-         [AC_MSG_WARN([No PMI found, if build fails consider --enable-pmi-simple or --with-pmi])])
+         [AC_MSG_ERROR([No PMI client interface was configured])])
```

Comment:
What's the rationale for removing `consider --enable-pmi-simple...`?  Seems like it's pretty helpful for noobs.